### PR TITLE
[15.0][IMP] partner_risk_insurance: Performance installation on large databases

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa 

ping @carlosdauden @Shide @rafaelbn

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: partner_risk_insurance/migrations/15.0.3.0.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    """Create account move columns to performance upgrade on large databases"""
+    openupgrade.logged_query(
+        env.cr,
+        """
+    ALTER TABLE account_move
+        ADD COLUMN IF NOT EXISTS insured_with_credit_policy boolean DEFAULT false,
+        ADD COLUMN IF NOT EXISTS credit_policy_company_id INT,
+        ADD COLUMN IF NOT EXISTS credit_policy_state_id INT;
+    """,
+    )
```
Comment (pedrobaeza) [inline partner_risk_insurance/migrations/15.0.3.0.1/pre-migration.py]: You don't need to repeat code:

```suggestion
from odoo.addons.partner_risk_insurance.hooks import pre_init_hook


def migrate(cr, version):
    """Create account move columns to performance upgrade on large databases"""
    pre_init_hook(cr)
```

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by Shide** [comment]
Thanks @sergio-teruel, @carlosdauden and @pedrobaeza for the fast PR. Sorry I can't do it yesterday 😢

**Pull request merged**